packages/next/docs/server-next-executor-examples.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Next server executor examples
+description: This page contains examples for the @nx/next:serve executor.
+---
+
+`project.json`:
+
+```json
+//...
+{
+  "name": "acme",
+  "$schema": "node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": ".",
+  "projectType": "application",
+  "targets": {
+    //...
+    "serve": {
+      "executor": "@nx/next:server",
+      "defaultConfiguration": "production",
+      "options": {
+        "buildTarget": "acme:build",
+        "dev": true
+      }
+    }
+    //...
+  }
+}
+```
+
+```bash
+nx run acme:serve
+```
+
+## Examples
+
+### For Next Standalone projects
+
+{% tabs %}
+{% tab label="Default configuration" %}
+
+This is the default configuration for Next standalone projects. Our `@nx/next:server` executor is integrated to use Next.js' CLI. You can read more about the serve options at [Next.js CLI Options](https://nextjs.org/docs/app/api-reference/next-cli)
+
+```json
+    "serve": {
+      "executor": "@nx/next:server",
+      "defaultConfiguration": "development",
+      "options": {
+        "buildTarget": "acme:build",
+        "dev": true
+      },
+      "configurations": {
+        "development": {
+          "buildTarget": "acme:build:development",
+          "dev": true
+        },
+        "production": {
+          "buildTarget": "acme:build:production",
+          "dev": false
+        }
+      }
+    },
+```
+
+{% /tab %}
+{% tab label="Enable turbo" %}
+
+Turbopack (beta) is a cutting-edge bundler designed for JavaScript and TypeScript, meticulously crafted in Rust, and seamlessly integrated within Next.js. To read more about support features see [Next.js Turbopack Doucmentation](https://turbo.build/pack/docs/features)
```

Comment:
I'd leave out the `, meticulously crafted in Rust, and seamlessly integrated within Next.js` part. Sounds too much like marketing buzzwords for Vercel. 

Just `Turbopack (beta) is a cutting-edge bundler designed for JavaScript and TypeScript. To read more about support features see...` is fine.